src/application/F3DOptionsParser.cxx

```diff
@@ -179,101 +174,101 @@ class ConfigurationOptions
 };
 
 //----------------------------------------------------------------------------
-F3DOptions ConfigurationOptions::GetOptionsFromArgs(std::vector<std::string>& inputs)
+void ConfigurationOptions::GetOptionsFromArgs(
+  F3DAppOptions& appOptions, f3d::options& options, std::vector<std::string>& inputs)
 {
-  F3DOptions options;
   try
   {
-    cxxopts::Options cxxOptions(f3d::AppName, f3d::AppTitle);
+    // TODO From where ?
+    cxxopts::Options cxxOptions("F3D", "F3D");
```

Comment:
Can you add back f3d::AppName and f3d::AppTitle in an application-side CMake generated file?